resources/user_config.ts

```diff
@@ -26,7 +26,7 @@ import { Responses as _Responses } from './responses';
 const Responses = _Responses;
 import _Outputs from './outputs';
 const Outputs = _Outputs;
-import _Util from './util';
+import * as _Util from './util';
```

Comment:
Would you mind changing this to be explicit imports? I think I should add eslint to not have `import *` for consistency.

Also, I think in this case, it would be better to only import the functions that used to exist and put them in a structure called `Util` so that we can only include functions in user triggers that we want to.

Reply:
Are you saying that should be changed to something like:
```ts
import {
  isTankJob,
  isHealerjob,
  // ...
} from './util';
const Util = {
  isTankJob,
  isHealerjob,
  // ...
};
```
?

Reply:
Yeah.  It's a little wordy, but I think it's better to be explicit than include all future additions to `Util`.